src/commons/availability/lead-time-calculator.js

```
@@ -0,0 +1,221 @@
+  }
+
+  return windows;
+}
+
+/**
+ * @param {import("../entities/bookable/bookable").Bookable|Object} bookable
+ * @param {number} timeBegin
+ * @param {Date} [now]
+ * @returns {boolean}
+ */
+function hasSufficientPreparationLeadTime(bookable, timeBegin, now = new Date()) {
```

ESLint

[error] Replace `bookable,·timeBegin,·now·=·new·Date()` with `⏎··bookable,⏎··timeBegin,⏎··now·=·new·Date(),⏎`

(prettier/prettier)